_ml-commons-plugin/agents-tools/output-processors.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: default
+title: Output processors
+parent: Agents and tools
+grand_parent: ML Commons APIs
+nav_order: 30
+---
+
+# Output processors
+**Introduced 3.3**
+{: .label .label-purple }
+
+Output processors allow you to modify and transform the output of any tool before it's returned to the agent or user. You can chain multiple output processors together to create complex data transformation pipelines that execute sequentially.
+
+## Overview
+
+Output processors provide a powerful way to:
+
+- **Transform data formats**: Convert between different data structures (strings, JSON, arrays)
+- **Extract specific information**: Use JSONPath or regex patterns to pull out relevant data
+- **Clean and filter content**: Remove unwanted fields or apply formatting rules
+- **Standardize outputs**: Ensure consistent data formats across different tools
+
+Each tool can have multiple output processors that execute in the order they are defined. The output of one processor becomes the input for the next processor in the chain.
+
+## Configuration
+
+Add output processors to any tool by including an `output_processors` array in the tool's `parameters` section during agent registeration:
+
+Example:
```

Comment:
Seems this is not a real example as the type is `processor_type`. Let's clarify this is a template ? Or use a real example?